packages/common/util/src/params.test.ts

```diff
@@ -0,0 +1,39 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import * as S from '@effect/schema/Schema';
+import { expect } from 'chai';
+
+import { test } from '@dxos/test';
```

Comment:
since `@dxos/util` is configured with vitest you should import those DSL methods from `vitest` from now on